connPFM/utils/io.py

```diff
@@ -63,12 +65,44 @@ def save_img(data, output, masker, history_str=None):
 
 
 def dask_scheduler(jobs):
-    config.set(distributed__comm__timeouts__tcp="90s")
-    config.set(distributed__comm__timeouts__connect="90s")
-    config.set(scheduler="single-threaded")
-    config.set({"distributed.scheduler.allowed-failures": 50})
-    config.set(admin__tick__limit="3h")
-    cluster = SGECluster(memory="20Gb")  # assumees dask configuration file
-    client = Client(cluster)
-    cluster.scale(jobs)
+    # look if default ~ .config/dask/jobqueue.yaml exitsts
+    with open(join(expanduser("~"), ".config/dask/jobqueue.yaml"), "r") as stream:
+        data = yaml.load(stream, Loader=yaml.FullLoader)
+    if data is None:
+        LGR.warning(
+            "dask configuration wasn't detected, "
+            "if you are using a cluster please look at "
+            "the jobqueue YAML example, modify it so it works in your cluster "
+            "and add it to ~/.config/dask "
+            "local configuration will be used"
+        )
+        cluster = None
+    else:
+        config.set(distributed__comm__timeouts__tcp="90s")
+        config.set(distributed__comm__timeouts__connect="90s")
+        config.set(scheduler="single-threaded")
+        config.set({"distributed.scheduler.allowed-failures": 50})
+        config.set(admin__tick__limit="3h")
+        if "sge" in data["jobqueue"]:
+            cluster = SGECluster(memory="20Gb")
```

Comment:
Shouldn't the memory be a parameter users input?

Reply:
In any case i would leave it empty and the user can modify this in the jobqueue.yaml file. I can actually add an example file

Reply:
I think having an example would be great.